devices/common_patches/h7_dbgmcu.yaml

```diff
@@ -0,0 +1,197 @@
+        fields:
+          WWDG1:
+            description: WWDG1 stop in debug mode
+            bitOffset: 6
+            bitWidth: 1
+      APB1LFZ1:
+        description: APB1L peripheral freeze register
+        addressOffset: 0x03C
+        access: read-write
+        resetValue: 0x00000000
+        fields:
+          I2C3:
+            description: I2C3 SMBUS timeout stop in debug mode
+            bitOffset: 23
+            bitWidth: 1
+          I2C2:
+            description: I2C2 SMBUS timeout stop in debug mode
+            bitOffset: 22
+            bitWidth: 1
+          I2C1:
+            description: I2C1 SMBUS timeout stop in debug mode
+            bitOffset: 21
+            bitWidth: 1
+          LPTIM1:
+            description: LPTIM1 stop in debug mode
+            bitOffset: 9
+            bitWidth: 1
+          TIM14:
+            description: TIM14 stop in debug mode
+            bitOffset: 8
+            bitWidth: 1
+          TIM13:
+            description: TIM13 stop in debug mode
+            bitOffset: 7
+            bitWidth: 1
+          TIM12:
+            description: TIM12 stop in debug mode
+            bitOffset: 6
+            bitWidth: 1
+          TIM7:
+            description: TIM7 stop in debug mode
+            bitOffset: 5
+            bitWidth: 1
+          TIM6:
+            description: TIM6 stop in debug mode
+            bitOffset: 4
+            bitWidth: 1
+          TIM5:
+            description: TIM5 stop in debug mode
+            bitOffset: 3
+            bitWidth: 1
+          TIM4:
+            description: TIM4 stop in debug mode
+            bitOffset: 2
+            bitWidth: 1
+          TIM3:
+            description: TIM3 stop in debug mode
+            bitOffset: 1
+            bitWidth: 1
+          TIM2:
+            description: TIM2 stop in debug mode
+            bitOffset: 0
+            bitWidth: 1
+      APB2FZ1:
+        description: APB2 peripheral freeze register
+        addressOffset: 0x04C
+        access: read-write
+        resetValue: 0x00000000
+        fields:
+          HRTIM:
+            description: HRTIM stop in debug mode
+            bitOffset: 29
+            bitWidth: 1
+          TIM17:
+            description: TIM17 stop in debug mode
+            bitOffset: 18
+            bitWidth: 1
+          TIM16:
+            description: TIM16 stop in debug mode
+            bitOffset: 17
+            bitWidth: 1
+          TIM15:
+            description: TIM15 stop in debug mode
+            bitOffset: 16
+            bitWidth: 1
+          TIM8:
+            description: TIM8 stop in debug mode
+            bitOffset: 1
+            bitWidth: 1
+          TIM1:
+            description: TIM1 stop in debug mode
+            bitOffset: 0
+            bitWidth: 1
+      APB4FZ1:
+        description: APB4 peripheral freeze register
+        addressOffset: 0x054
+        access: read-write
+        resetValue: 0x00000000
+        fields:
+          WDGLSD1:
```

Comment:
I see `DBG_IWDG1` in the RM page for this register (well they also spell it `DBG_IIWDG1` but I guess that's a typo), whereas the register map has it `WDGLSD1`, but the revision 6 changelog suggests it was renamed to `IWDG1` and I guess they forgot to update the map.

Reply:
I had actually used an older RM. Will change to `IWDG1`.